plugins/TwoFactorAuth/vue/src/UserSettings/UserSettings.vue

```diff
@@ -0,0 +1,143 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <ContentBlock
+    :content-title="contentTitle"
+    class="userSettings2FA"
+  >
+    <p v-html="twoFactorAuthIntro"></p>
```

Comment:
Isn't this required to have our tests passing?
```suggestion
    <p v-html="$sanitize(twoFactorAuthIntro)"></p>
```

Reply:
:+1: yes, must have missed this I guess